src/EventStore.Core/Services/Storage/StorageScavenger.cs

```diff
@@ -89,7 +89,7 @@ public class StorageScavenger :
 					} else {
 						message.Envelope.ReplyWith(new ClientMessage.ScavengeDatabaseResponse(message.CorrelationId,
 							ClientMessage.ScavengeDatabaseResponse.ScavengeResult.InvalidScavengeId,
-							_currentScavenge?.ScavengeId));
```

Comment:
Is there any particular reason why `currentScavenge.ScavengId` was changed to the `message.ScavengeId`?

Reply:
Yes, I would like to report back which scavengeid was invalid.

Reply:
Judging by the code around this, it looks like the original purpose was to return the currently running scavenge Id so if you sent the wrong one, you'd be able to get the correct one and be able to stop it in a subsequent call

Reply:
ok, makes sense. fixed now